x-pack/plugin/security/qa/multi-cluster/src/javaRestTest/java/org/elasticsearch/xpack/remotecluster/RemoteClusterSecurityEsqlIT.java

```diff
@@ -297,7 +300,15 @@ public void testCrossClusterQuery() throws Exception {
                     | LIMIT 2
                     | KEEP emp_id, department"""));
                 assertOK(response);
-                Map<String, Object> values = entityAsMap(response);
+                Map<String, Object> responseAsMap = entityAsMap(response);
+                List<?> columns = (List<?>) responseAsMap.get("columns");
+                List<?> values = (List<?>) responseAsMap.get("values");
```

Comment:
nit: we could refactor and move flattening and assertion into a dedicated method and reuse it in other tests below

Reply:
done in https://github.com/elastic/elasticsearch/pull/107079/commits/02e1c25856e8bd9996f58128fbc4921711a82eb0